Motivate aliasing rules for non-iso, non-trn (#378)
ponylang/pony-tutorial: content/reference-capabilities/aliasing.md

  var b: Wombat box = a // Allowed!
```

> __What about aliasing other stuff?__ For both `iso` and `trn`, the guarantees impose aliases to give up on some ability (reading and writing for `iso`, writing for `trn`). For the other capabilities `ref`, `val`, `box` and `tag`, aliases allow for the same operations, so a reference can just be aliased as itself.

Reviewer: ```suggestion
__What about aliasing other stuff?__ For both `iso` and `trn`, the guarantees require that aliases must give up on some ability (reading and writing for `iso`, writing for `trn`). For the other capabilities (`ref`, `val`, `box` and `tag`), aliases allow for the same operations, so such a reference can just be aliased as itself.
```
